ui/app/components/auth-saml.hbs

```diff
@@ -22,8 +22,7 @@
         />
       </div>
       <AlertInline
-        @sizeSmall={{true}}
```

Comment:
the class `size-small` no longer exists (functionality have been removed previously?), so remove this arg all together.